packages/renderer/src/lib/container/ContainerColumnName.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+import { router } from 'tinro';
+
+import type { ContainerGroupInfoUI, ContainerInfoUI } from './ContainerInfoUI';
+import { ContainerGroupInfoTypeUI } from './ContainerInfoUI';
+
+export let object: ContainerInfoUI | ContainerGroupInfoUI;
+
+function displayContainersCount(containerGroup: ContainerGroupInfoUI) {
+  let result = containerGroup.allContainersCount + ' container' + (containerGroup.allContainersCount > 1 ? 's' : '');
+  if (containerGroup.containers.length !== containerGroup.allContainersCount) {
+    result += ` (${containerGroup.allContainersCount - containerGroup.containers.length} filtered)`;
+  }
+  return result;
+}
+
+function openGroupDetails(containerGroup: ContainerGroupInfoUI) {
+  if (!containerGroup.engineId) {
+    return;
+  }
+  if (containerGroup.type === ContainerGroupInfoTypeUI.POD) {
+    router.goto(`/pods/podman/${encodeURI(containerGroup.name)}/${encodeURIComponent(containerGroup.engineId)}/logs`);
```

Comment:
I am wondering if we should not uses the `handleNavigation` from 

https://github.com/containers/podman-desktop/blob/54cdb5cd43cae0162f533221a1f4e0bd74dce66a/packages/renderer/src/navigation.ts#L32

to avoid hardcoding the urls in the code in multiple places

> Not a blocker

Reply:
Yes, we should do this. I opened #4966 a while ago, I think we should target a separate PR (or PRs) to clean this up in other places as well.